src/CreateSourceEmulator.hpp

```diff
@@ -50,22 +50,27 @@ createSourceEmulator(const appfwk::app::ConnectionReference qi, std::atomic<bool
   static constexpr int daphne_time_tick_diff = 16;
   static constexpr double daphne_dropout_rate = 0.9;
   static constexpr double daphne_rate_khz = 200.0;
+  static constexpr int daphne_frames_per_tick = 1;
 
   static constexpr int wib_time_tick_diff = 25;
   static constexpr double wib_dropout_rate = 0.0;
   static constexpr double wib_rate_khz = 166.0;
+  static constexpr int wib_frames_per_tick = 1;
 
   static constexpr int wib2_time_tick_diff = 32;
   static constexpr double wib2_dropout_rate = 0.0;
   static constexpr double wib2_rate_khz = 166.0;
+  static constexpr int wib2_frames_per_tick = 1;
 
   static constexpr int wibeth_time_tick_diff = 32*64;
   static constexpr double wibeth_dropout_rate = 0.0;
   static constexpr double wibeth_rate_khz = 30.5176;
+  static constexpr int wibeth_frames_per_tick = 1;
 
-  static constexpr int tde_time_tick_diff = 1000;
+  static constexpr int tde_time_tick_diff = 32*4474;
```

Comment:
As a first occasion of best practice, should this field be updated to the tick difference from fdreadoutlibs? Maybe in one go we should update the rest?